addons/cert-manager/addon.rb

```diff
@@ -50,6 +50,11 @@ def self.messages
   }
 
   install {
+    # Need to add label to kube-system NS to get webhook PKI in place properly
+    kube_system = kube_client.api('v1').resource('namespaces').get('kube-system')
+    kube_system.metadata.labels['certmanager.k8s.io/disable-validation'] = "true"
+    kube_client.api('v1').resource('namespaces').update_resource(kube_system)
```

Comment:
Why update instead of patch?

Reply:
I remembered that nodes didn't support patching but now thinking of it I might be wrong :)

Reply:
Yeah, should use patch. will fix